src/a09-dates/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "a09-dates",
+  "version": "1.0.0",
+  "description": "",
+  "type": "module",
+  "main": "index.js",
+  "scripts": {
+    "test": "node --test",
+    "solution": "node --test ./test/solution.test.js",
+    "verify": " NODE_NO_WARNINGS=1 node --test --loader=../../verify/loader.js ./verifiers/handler.js"
```

Comment:
specifying env vars like this will not work on windows. why do we need to provide env vars in the first place btw?

Reply:
Uh, I wasn't aware that this doesn't work on Win!

Without this var, we have a warning from the console which tells us that `loader` is experimental.
This variable suppresses that warning (we don't want to show that to the user since the usage of the loader is internal in the verify feature, ideally should be transparent to the user)